airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/kotlin/io/airbyte/integrations/base/destination/typing_deduping/PostImportAction.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.destination.typing_deduping
+
+enum class PostImportAction {
+    DO_NOTHING,
+    DEDUPE,
+}
```

Comment:
Nice. earlier we did care about `DestinationSyncMode.OVERWRITE` but now it is truncate sync for all intents and this simplifies the 2 variants we care for dedupe.

Reply:
The OVERWRITE was super dangerous because one of the 1st thing we did was if (mode == OVERWRITE) {mode = APPEND}, so OVERWRITE wasn't doing anything anymore